frontend/src/pages/Landing/index.js

```diff
@@ -40,16 +41,21 @@ export function renderTotal(offset, perPage, activePage, reportsCount) {
   return `${from}-${to} of ${reportsCount}`;
 }
 
-function regionFilter(regionId) {
-  return {
-    topic: 'region',
-    condition: 'Contains',
-    query: regionId.toString(),
-  };
-}
-
 function Landing({ user }) {
+  // Determine Default Region.
   const regions = allRegionsUserHasPermissionTo(user);
+  const regionToUse = regions[0] || user.homeRegionId || 0;
+  const defaultRegion = user.homeRegionId === 14 ? 14 : regionToUse;
```

Comment:
What are all the checks for region 14 when there are only 12 regions?

Reply:
Region 14 is all regions.